contrib/errorprone/README.md

```diff
@@ -0,0 +1,60 @@
+# Pants plugin for Error Prone
+
+The Error Prone plugin runs static analysis on Java source files and looks for various
+[bug patterns](http://errorprone.info/bugpatterns) not reported by the standard javac compiler.
+
+A full description of Error Prone can be found on the [Error Prone web page](http://errorprone.info/).
+
+The version of Error Prone used by the plugin requires Java 8 to run.  If you want to run Error Prone with Java 7 you will need to use version 2.0.5 or earlier. Using Java 7 may require changes to the bootclasspath to override certain classes from `rt.jar`.  See https://github.com/google/error-prone/issues/287 for more info.
+
+One notable difference between this plugin and other Error Prone integrations is this plugin does not replace the javac executable used for compilation.  Instead, it runs the Error Prone javac as a seperate task after regular javac compilation is run.  This is obviously slower than running the compiler once but it avoids a trickier integration with Pants incremental compilation tool [Zinc](https://github.com/sbt/zinc).
+
+
+## Installation
+
+Error Prone support is provided by a plugin distributed to [pypi]
+(https://pypi.python.org/pypi/pantsbuild.pants.contrib.errorprone).
+Assuming you have already [installed pants](http://www.pantsbuild.org/install.html), you'll need to
+add the Error Prone plugin in your `pants.ini`, like so:
+```ini
+[GLOBAL]
+pants_version: 1.3.0
+
+plugins: [
+    'pantsbuild.pants.contrib.errorprone==%(pants_version)s',
+  ]
+```
+
+## Running
+
+When you run `./pants compile` Error Prone is executed after the compile step and will run on any targets that contain java files.
+
+```
+./pants compile <target>
+...
+      00:07:42 00:00     [compile]
+      00:07:42 00:00     [zinc]
+      00:07:42 00:00     [jvm-dep-check]
+      00:07:42 00:00     [checkstyle]
+      00:07:42 00:00     [errorprone]
+                         Invalidated 7 targets.
+      00:07:42 00:00       [errorprone]
+...
+```
+
```

Comment:
Are you thinking I should add something to [Common Tasks](http://www.pantsbuild.org/common_tasks.html) or start a new category for Contrib Tasks?